graylog2-web-interface/src/views/logic/fieldactions/ChangeFieldType/ChangeFieldTypeModal.tsx

```diff
@@ -132,13 +135,16 @@ const ChangeFieldTypeModal = ({ show, onSubmitCallback, fieldTypes, isOptionsLoa
                         show={show}
```

Comment:
It wold be nice to display a loading indicator for the button while the type is being changes / removed.

Reply:
I will just disable it. Looks like we don't have these loader options for modal form component

Reply:
In addition to disabling it, we could do something like: `submitButtonText={loading ? 'Changing field type...' : 'Change field type'}`.

It would be nice to unify the behaviour with (most) other modals.